### PR TITLE
Clean up general Storybook structure and docs pages

### PR DESCRIPTION
**Pull request by ArnaudWeyts**
### Description

- Simplified the introduction page and added links to zeroheight, figma and github
- Simplified colors docs
- Removed Depths page
- Moved Illustrations page to lower level blocks, added installation instructions
- Added installation instructions for Icons
- Canvas / Docs tabs will no longer be shown in production (only in development), docs is the default
- Added Changelog page

### CODE REVIEW

File: .storybook/stories/changelog.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Meta, Description } from '@storybook/addon-docs';
+
+import changelog from '../../CHANGELOG.md';
+
+<Meta title="Changelog" />
```
Comment (lorgan3) [inline .storybook/stories/changelog.stories.mdx:5]: Can you use the constant here? The sorting wouldn't work if these were different
```suggestion
<Meta title={CHANGELOG} />
```

**Review by lowiebenoot — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]
Looks like some great improvements!

**Review by sanderbrugge — APPROVED** [review]

File: .storybook/stories/changelog.stories.mdx (hunk 1 above)
Comment (ArnaudWeyts) [reply .storybook/stories/changelog.stories.mdx:5]: Ah, you're right, but the constants are also not used in the other mdx files, I'll create a follow-up for it if you don't mind :)

**Comment by qubis741** [comment]
From ticket: 

- "Introduction should be one page Should include a link to Github, Zeroheight and Figma" 
  - Zeroheight == https://ahoy.teamleader.design/3547946e7/p/223d3f-ahoy ?

- "Typography page should be more customizable" 
  - I dont see any customizable options

**Comment by ArnaudWeyts** [comment]
> From ticket:
> 
>     * "Introduction should be one page Should include a link to Github, Zeroheight and Figma"
>       
>       * Zeroheight == https://ahoy.teamleader.design/3547946e7/p/223d3f-ahoy ?
> 
>     * "Typography page should be more customizable"
>       
>       * I dont see any customizable options

@qubis741 Yep, zeroheight is indeed that link. It's the platform our design system uses to deploy the guidelines.

About the typography guidelines, I've spent quite some time on trying to fix the inputs, but didn't manage to come to a solution. I noticed we have other components that are experiencing these issues as well, so I'll take it up in a separate ticket

**Review by qubis741 — APPROVED** [review]

**Pull request merged**